pandas/plotting/_core.py

```diff
@@ -875,6 +886,71 @@ def _make_plot(self):
             ax.errorbar(data[x].values, data[y].values,
                         linestyle='none', **err_kwds)
 
+    def _sci_notation(self, num):
+        '''
+        Returns mantissa and exponent of the number passed in agument.
+        Example:
+        _sci_notation(89278.8924)
```

Comment:
this can just be a module level function

Reply:
Does matplotlib have anything that does this? (cc @tacaswell)

Reply:
Also, typically the docstring formatting is like

```python
>>> _sci_notation(89278.8924)
(8.9, 5.0)
```

identical to what you get from the regular python REPL.

Reply:
Yes, but buried in the formatter code. I would not try to re-use it.

Reply:
Should we place this function as a module level function as suggested by @jreback or is there another module that would be more appropriate ? Maybe plotting/_converter.py ?

Reply:
module level in `plotting/_core.py` is fine I think.